wagtail/users/tests/test_utils.py

```diff
@@ -0,0 +1,74 @@
+from django.test import TestCase, override_settings
+
+from wagtail.users.utils import get_gravatar_url
+
+
+class TestGravatar(TestCase):
+    def test_gravatar_default(self):
+        # Test with the default settings
```

Comment:
```suggestion
        """ Test with the default settings. """
```

This is not critical but it might be nice to use the triple quotations for test docs, it makes it a bit easier to read and while we do not use this consistently in the unit tests it's nice to write new tests with this approach.